Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
What will be a good doc comment for the before class and the `$this->app->before...` line?